### DISCUSSION TITLE
How to make different color between the pointer？

### DISCUSSION THREAD

**Discussion by cjwn** [zepp-health/zeppos-forum#319] [Q&A]
Hello everyone! How is the different color between the hour and minute hands in the watchface specification achieved? I've looked at the documentation's widgets, and it seems that circle, arc, and img cannot implement this style.

<img width="640" alt="image" src="https://github.com/zepp-health/zeppos-forum/assets/7097603/9ae7a366-2fe1-4bac-b9d3-0a4a6445a52d">

**Comment by MyoungXUE** [comment]
Hello, this issue has been resolved in the Lark group. I comment here so that this answer can help more developers.

Use the [ARC](https://docs.zepp.com/docs/watchface/api/hmUI/widget/ARC/) widget, set the `line_width` to half the width of the device, and calculate the angle based on the current time.
